test/specs/elements/Button/Button-test.js

```diff
@@ -116,6 +116,16 @@ describe('Button', () => {
       shallow(<Button label='hi' />)
         .should.have.className('labeled')
     })
+    it('adds the disabled className to the root element only when the disabled attribute is set', () => {
```

Comment:
This test is covered by `common.propKeyOnlyToClassName(Button, 'disabled')` which asserts the prop is converted into a className.  We can remove the test here.